setup.py

```diff
@@ -64,7 +64,7 @@ def get_requirements(filename: str) -> List[str]:
         "License :: OSI Approved :: Apache Software License",
     ],
     packages=setuptools.find_packages(),
-    python_requires=">=3.9",
+    python_requires=">=3.9,<3.12",
```

Comment:
Are you sure 3.12 works? Last time I tried there were no datumaro package for this version

Reply:
No, it's 'less than 3.12', so it doesn't include 3.12

Reply:
You are right 😁